util/version/version.go

```diff
@@ -74,6 +75,14 @@ func String() string {
 	return version
 }
 
+// BuildVersion returns the repository version information from where
+// politeia was built.
+func BuildVersion() string {
+	info, _ := debug.ReadBuildInfo()
+	version := info.Main.Version
+	return version
```

Comment:
No need to create a new variable.  Just do.

```suggestion
	return info.Main.Version
```